tests/integration/targets/inventory_kubevirt/playbook.yml

```diff
@@ -0,0 +1,62 @@
+- name: Create VM
+  connection: local
+  gather_facts: false
+  hosts: localhost
+  tasks:
+    - name: Create a VirtualMachine
+      kubernetes.kubevirt.kubevirt_vm:
+        state: present
+        name: testvm
+        namespace: default
+        labels:
+          app: test
+        wait: yes
+        wait_timeout: 600
```

Comment:
Default of 300 does not suffice?

Reply:
I tried to increase the timeout to see if it was only being slow, but I am afraid there are intermittent boot failures

Reply:
Can we revert to 300 or should we keep 600 after the change to Centos?

Reply:
lets keep 600 in case when github actions is on a bad day